src/sage/rings/polynomial/multi_polynomial_ideal.py

```diff
@@ -3380,26 +3429,32 @@ def __init__(self, ring, gens, coerce=True, side = "left"):
         - ``gens`` - the generators of this ideal
         - ``coerce`` (optional - default True) - generators are
           coerced into the ring before creating the ideal
-        - ``side`` - optional string, either "left" (default)
-          or "twosided"; defines whether this ideal is left
+        - ``side`` - optional string, either ``"left"`` (default)
+          or ``"twosided"``; defines whether this ideal is left
           of two-sided.
 
         EXAMPLES::
 
-            sage: A.<x,y,z> = FreeAlgebra(QQ, 3)
-            sage: H = A.g_algebra({y*x:x*y-z, z*x:x*z+2*x, z*y:y*z-2*y})
-            sage: H.inject_variables()
+            sage: A.<x,y,z> = FreeAlgebra(QQ, 3)                                        # optional - sage.combinat sage.modules
+            sage: H = A.g_algebra({y*x: x*y-z, z*x: x*z+2*x, z*y: y*z-2*y})             # optional - sage.combinat sage.modules
+            sage: H.inject_variables()                                                  # optional - sage.combinat sage.modules
             Defining x, y, z
```

Comment:
This is assuming that the algebras defined in `sage.algebras` will be shipped in a distribution alongside `sage.combinat` -- "because their bases are defined combinatorially"; but I don't have a more detailed plan than that at the moment.